sql/hive/src/test/scala/org/apache/spark/sql/hive/QueryPartitionSuite.scala

```diff
@@ -52,8 +52,9 @@ class QueryPartitionSuite extends QueryTest {
         ++ testData.toSchemaRDD.collect ++ testData.toSchemaRDD.collect)
 
     // delete the path of one partition
-    val folders = tmpDir.listFiles.filter(_.isDirectory)
-    Utils.deleteRecursively(folders(0))
+    tmpDir.listFiles
```

Comment:
LGTM. Since `toSchemaRDD` isn't relevant to this change it can probably be dealt with separately and all in one go.